scripts/automation/style/pep8.py

```diff
@@ -0,0 +1,29 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+def fix_pip8(directory):
```

Comment:
typo: should be `fix_pep8`.  Also, where does it "fit" (i.e. how and when will we invoke it?) Should it be added to the CI scripts?